gridcapa-swe-runner-app/src/test/java/com/farao_community/farao/swe/runner/app/services/FileExporterTest.java

```diff
@@ -144,4 +148,20 @@ void cgmFileNameTest() {
         String cgmFileName = fileExporter.getCgmZipFileName(OffsetDateTime.parse("2023-01-01T00:30Z"), DichotomyDirection.ES_FR);
         assertEquals("20230101_0130_CGM_ESFR.zip", cgmFileName);
     }
+
+    @Test
+    void exportCgmesZipFileTest() {
+        SweData sweData = Mockito.mock(SweData.class);
+        Mockito.when(sweData.getTimestamp()).thenReturn(dateTime);
+        Mockito.when(minioAdapter.generatePreSignedUrl("2021/04/01/23_30/OUTPUTS/20210401_2330_CGM_PTES.zip")).thenReturn("SUCCESS");
+        Map<String, ByteArrayOutputStream> inputFiles = new HashMap<>();
+        inputFiles.put("firstFile", new ByteArrayOutputStream());
+        inputFiles.put("secondFile", new ByteArrayOutputStream());
+        inputFiles.put("thirdFile", new ByteArrayOutputStream());
+        try {
+            assertEquals("SUCCESS", fileExporter.exportCgmesZipFile(sweData, inputFiles, DichotomyDirection.PT_ES, "CGM_PTES"));
+        } catch (Exception e) {
```

Comment:
Why do you add a catch-fail here, rather than just letting the exception be raised at upper level? If an exception occurs in your test, and the test was not designed to check that this exception would happen, it surely means that the exception should not be thrown... Therefore I don't really see the point in explicitly failing.

Reply:
ok